api/account.go

```diff
@@ -18,6 +18,10 @@ type (
 		// Balance is the balance of the account.
 		Balance *big.Int `json:"balance"`
 
+		// Drift is the accumulated delta between the bus' tracked balance for
+		// an account and the balance reported by a host.
+		Drift *big.Int `json:"drift"`
```

Comment:
We are not losing data since refilling doesn't reset the drift. But regardless this feels like a metrics thing and can probably be tracked and visualised by something like Prometheus or ELK. That will show you resets of the balance and the time it took to accumulate a lot better than tracking another value in renterd.

Reply:
Isn't it odd that we have a `maxDrift` cap and at the same time never reset the `Drift` field. You are essentially giving the account a lifetime where it will eventually run into the `maxDrift` threshold and cause issues. Unless of course drift is 0 but that's an assumption we can't make I think since drift is unavoidable at times.

In my head there were two fields, one that keeps track of historical drift, which you can use (and keep using) to see how a host behaves in relation to other hosts, as well as a drift field that keeps track of drift since the last refill and thus gets reset when we sync.

Fine to resolve though, I'm just very weary of preventing refills